### PR TITLE
Sync labels

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

File: config.toml
Comment (ErikSchierboom) [inline config.toml:4]: @SaschaMann Is this how the path should be defined, using forward slashes?

File: synced-files/.github/labels.yml
Comment (ErikSchierboom) [inline synced-files/.github/labels.yml:1]: @iHiD I've added comments to help explain what the categories are for. Track maintainers will see this in their generated `.github/labels.yml` file, although they won't see it in their labels of course.

I've also re-ordered the labels and colors to have them progress from light to dark, and replaced the one colors that rendered with black text with the one color I had not yet used but which _did_ render as white text.

**Comment by ErikSchierboom** [comment]
@SaschaMann Once this is merged, will the workflow automatically trigger? I'd like to test the labels with one other track first if possible.

**Comment by SaschaMann** [comment]
Yes. You can use the workflow dispatch trigger to run it on this branch and give it a list of tracks for testing (feel free to use Julia for testing, too).

File: config.toml
Comment (SaschaMann) [reply config.toml:4]: Yes, that should work (haven't tested but don't see why not).

Perhaps add a trailing comma so the next diff is cleaner?

File: synced-files/.github/labels.yml
Comment (SaschaMann) [inline synced-files/.github/labels.yml:6]: Could you add a brief explanation that people can add labels via an append? I want to avoid people editing these directly and for now the file isn't code owner-protected (in the long run it probably should be)

File: synced-files/.github/workflows/sync-labels.yml
Comment (SaschaMann) [inline synced-files/.github/workflows/sync-labels.yml:19]: Since dependabot updates these but doesn't update the comments, I tend to leave out the comments nowadays. Otherwise they might get out of sync easily.

Comment (ErikSchierboom) [reply synced-files/.github/workflows/sync-labels.yml:19]: Yeah that makes sense. I'll update

File: synced-files/.github/labels.yml
Comment (ErikSchierboom) [reply synced-files/.github/labels.yml]: Done. Could you see if the new explanation makes sense?

File: synced-files/.github/labels.yml
Comment (iHiD) [inline synced-files/.github/labels.yml:15]: I suggest a divider here to seperate the general info above from the comment below. It confused me as they flowed together.

```suggestion
# ---
#
# These are the Exercism-wide labels which are shared across all repos.
```

**Review by iHiD — APPROVED** [review]

File: synced-files/.github/labels.yml
Comment (SaschaMann) [inline synced-files/.github/labels.yml:3]:
```suggestion
# This file is automatically generated by merging two files:
```

File: synced-files/.github/labels.yml
Comment (SaschaMann) [inline synced-files/.github/labels.yml:13]:
```suggestion
# Any changes to the repo's `.appends/.github/labels.yml`
# file or the Exercism-wide `labels.yml` file will automatically result in this
# file being regenerated and the labels being updated. This typically takes
# 5-10 minutes.
```

File: synced-files/.github/labels.yml
Comment (SaschaMann) [reply synced-files/.github/labels.yml]: It does, I left two small suggestions to clarify some things.

File: synced-files/.github/labels.yml
Comment (iHiD) [reply synced-files/.github/labels.yml:13]: What line-widths are we aiming for here? Feels like we could definitely compress this into at least 3 lines :)

Comment (SaschaMann) [reply synced-files/.github/labels.yml:13]: I have no idea, I tried to follow Erik's line width :D

Comment (iHiD) [reply synced-files/.github/labels.yml:13]: :D @ErikSchierboom? :)

Comment (iHiD) [reply synced-files/.github/labels.yml:13]: I've updated your suggestion to spread it onto three lines :)

Comment (ErikSchierboom) [reply synced-files/.github/labels.yml:13]: No particular line width

File: config.toml
Comment (ErikSchierboom) [reply config.toml]: Done

**Review by SaschaMann — APPROVED** [review]
Looks good to me. Have you tested it with an append present in the track, too?

**Comment by ErikSchierboom** [comment]
It seems to work: https://github.com/exercism/org-wide-files/actions/runs/906431356, https://github.com/exercism/csharp/pull/1534

**Review by SaschaMann — APPROVED** [review]
LGTM

File: global-files/.github/workflows/sync-labels.yml
Comment (SaschaMann) [inline global-files/.github/workflows/sync-labels.yml:15]: This should be changed then

**Pull request merged**
